src/app/pages/questions/components/question/timed-test/timed-test.component.ts

```diff
@@ -26,33 +32,32 @@ export class TimedTestComponent implements OnInit, OnChanges {
   timer: Timer
   @Input()
   currentlyShown: boolean
+
   public taskTimer: TaskTimer
+  startTime: number
+  endTime: number
 
-  constructor(private dialogs: Dialogs, private vibration: Vibration) {}
+  constructor(
+    private dialogs: Dialogs,
+    private vibration: Vibration,
+    private background: BackgroundMode
+  ) {}
 
   ngOnInit() {
+    this.background.enable()
```

Comment:
Do you mean when they click the home button or back button?

Reply:
Ok Sorry, maybe i am confusing this. I thought this will put the app in background automatically.
But if this is just so that app can work in background but will still require user to press home button to move it to background then it's ok.

Reply:
Yup the user would have to press the home button to move to the background.